<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConfiguration</class>
 <widget class="QWidget" name="FormConfiguration">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Configuration setup</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>581</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Properties</string>
   </property>
   <widget class="QFrame" name="frameBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxControls">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QPushButton" name="pushButtonNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveAs">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveAs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>25</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguration</receiver>
   <slot>slotPressClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNew</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguration</receiver>
   <slot>slotPressNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguration</receiver>
   <slot>slotPressLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguration</receiver>
   <slot>slotPressSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>FormConfiguration</receiver>
   <slot>slotPressSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCloseLauncher()</slot>
  <slot>window_loaded()</slot>
  <slot>slotPressNew()</slot>
  <slot>slotPressLoad()</slot>
  <slot>slotPressSave()</slot>
  <slot>slotPressSaveAs()</slot>
  <slot>slotPressClose()</slot>
 </slots>
</ui>
